src/lib.rs

```diff
@@ -45,6 +46,7 @@ pub fn args(year: u16) -> App<'static> {
                 .long("bench")
                 .about("Run criterion benchmarks"),
         )
+        .arg(Arg::new("all").long("all").about("Run all days"))
```

Comment:
It would be nice to add a short `-a` :) Maybe it should also conflict with `days`?